javascript/401-code-challenges/trees/__tests__/max-value.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+const { BinaryTree, Node } = require('../index');
+
+
+describe('BinaryTree findMaxValue function', () => {
+  test('Can successfully return the max value from a binary tree', () => {
+    const tree = new BinaryTree();
+    tree.root = new Node(1);
+    tree.root.left = new Node(5);
+    tree.root.right = new Node(9);
+    tree.root.left.left = new Node(7);
+    tree.root.left.right = new Node(3);
+
+    expect(tree.findMaxValue()).toEqual(9);
+  });
```

Comment:
The test doesn't reveal the problem , because the largest number just happens to be all the way right.